### PR TITLE
feat(version): throw error when webpack 2 is installed

### LINKED ISSUE #152: Add a check for the version of webpack
If the version is 2, then we can log a warning indicating that it's not supported and should not be used. Thoughts?

### PR DESCRIPTION
**Pull request by kentcdodds**
I think I'll just merge this if the build passes.

Closes #152

BREAKING CHANGE: webpack v2 is not supported and now there will be a
message when you're using webpack v2 with webpack-validator.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,9 +81,9 @@
     "semantic-release": "^4.3.5",
     "sinon": "1.17.3",
     "validate-commit-msg": "2.6.1",
-    "webpack": "*",
-    "webpack-md5-hash": "*",
-    "webpack-notifier": "*",
+    "webpack": "^1",
```
Comment (kentcdodds) [inline package.json]: Our build was breaking due to all of these `*` version ranges.

**Comment by kentcdodds** [comment]
I should probably say that my attitude toward this library is very much: "Let's stop maintaining this." I don't want people using this thing anymore. If it doesn't work for someone's use-case then they can fork. Let's all just move on.

**Comment by kentcdodds** [comment]
I'll go ahead and wait for someone to merge this.

**Comment by bebraw** [comment]
Yeah. I think the sanest option is to release a final version that's webpack 1 only and state clearly the package has been discontinued since webpack 2 implements the same functionality. I doubt there's a lot of interest in maintaining this package anymore. It would be more worthwhile to improve webpack core validation instead.

**Comment by kentcdodds** [comment]
Agreed. I'll merge this and a major version will be published which wont work with webpack 2 at all, so people will know. Thanks everyone for working on this! It was a good run :D

**Comment by jonathanewerner** [comment]
👍🏻 totally agreed.

**Pull request merged**